ui/src/main/java/org/eclipse/starter/ui/Project.java

```diff
@@ -308,7 +305,7 @@ public void generate() {
 						entry("profile", profile), entry("javaVersion", javaVersion),
 						entry("docker", (docker ? "yes" : "no")), entry("runtime", runtime)));
 
-				MavenUtility.invokeMavenArchetype("org.eclipse.starter", "jakarta-starter", ARCHETYPE_VERSION,
+				MavenUtility.invokeMavenArchetype("org.eclipse.starter", "jakarta-starter", VersionInfo.ARCHETYPE_VERSION,
```

Comment:
hummm, the coupling between these two classes makes me feel weird. Maybe an Util class with a static value that both classes can access would be better? It seems to me that VersionInfo is doing too much but this is a minor thing we can improve later



I will merge this one by now because it adds value